applications/GeoMechanicsApplication/python_scripts/geomechanics_analysis.py

```diff
@@ -60,6 +60,17 @@ def _CalculateTotalDisplacement(self,node, old_total_displacement):
         total_displacement = old_total_displacement + stage_displacement
         node.SetSolutionStepValue(KratosGeo.TOTAL_DISPLACEMENT, total_displacement)
 
+    def _CalculateIncrementalDisplacement(self, node):
+        """
+        Calculates incremental displacement
+        :param node:
+        :return:
+        """
+        incremental_displacement = node.GetSolutionStepValue(KratosMultiphysics.DISPLACEMENT, 0) - \
+                                   node.GetSolutionStepValue(KratosMultiphysics.DISPLACEMENT, 1)
```

Comment:
In general, I would try to avoid using the line continuation marker. You can do that as follows:
```suggestion
        incremental_displacement = (node.GetSolutionStepValue(KratosMultiphysics.DISPLACEMENT, 0) -
                                   node.GetSolutionStepValue(KratosMultiphysics.DISPLACEMENT, 1))
```

Reply:
Done